app/components/avo/views/resource_edit_component.rb

```diff
@@ -9,6 +9,14 @@ class Avo::Views::ResourceEditComponent < Avo::ResourceComponent
   prop :view, default: Avo::ViewInquirer.new(:edit).freeze
   prop :display_breadcrumbs, default: true, reader: :public
 
+  attr_reader :query
+
+  def initialize(resource:, query: nil, prefilled_fields: nil, **args)
+    @query = query
+    @prefilled_fields = prefilled_fields
+    super(resource: resource, **args)
+  end
```

Comment:
```suggestion
  prop :query
  prop :prefilled_fields
```

Lets use the props instead initializer